packages/wonder-blocks-dropdown/components/dropdown-launcher.js

```diff
@@ -207,6 +212,45 @@ export default class DropdownLauncher extends React.Component<Props, State> {
             </span>
         );
 
+        /*         TODO: Enforce better type check here
+        Currently this only looks for the 'icon' prop to decide
+        if an element is a IconButton or not as the 'isClassOf' method isnt present
+        */
+        if (childAnchor.props.icon) {
+            opener = (
```

Comment:
I'm not completely sure, but it seems this opener and the opener on line 206 use the same elements. If that's the case, this line can be removed.

Reply:
they do, this was in order to fix a flow error I was getting at the time